File: app/src/main/java/com/constraint/vagabond/data/RecAreaMedia.java
package com.constraint.vagabond.data;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public final class RecAreaMedia {

  @SerializedName("Title")
  @Expose
  public final String imageTitle;

  @SerializedName("Height")
  @Expose
  public final Integer imageHeight;

  @SerializedName("Width")
  @Expose
  public final Integer imageWidth;

  @SerializedName("URL")
  @Expose
  public final String imageURL;

  public RecAreaMedia(String imageTitle, Integer imageHeight, Integer imageWidth, String imageURL) {

    this.imageTitle = imageTitle;
    this.imageHeight = imageHeight;
    this.imageWidth = imageWidth;
    this.imageURL = imageURL;
  }
}

File: app/src/main/java/com/sierra/vagabond/data/entities/RecAreaMedia.kt
package com.sierra.vagabond.data.entities

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.sierra.vagabond.utils.IMG_HEIGHT
import com.sierra.vagabond.utils.IMG_TITLE
import com.sierra.vagabond.utils.IMG_URL
import com.sierra.vagabond.utils.IMG_WIDTH

data class RecAreaMedia(@field:SerializedName(IMG_TITLE) @field:Expose val imageTitle: String,
                        @field:SerializedName(IMG_HEIGHT) @field:Expose val imageHeight: Int,
                        @field:SerializedName(IMG_WIDTH) @field:Expose val imageWidth: Int,
                        @field:SerializedName(IMG_URL) @field:Expose val imageURL: String)
